datafusion-examples/examples/optimizer_rule.rs

```diff
@@ -100,6 +100,7 @@ pub async fn main() -> Result<()> {
 
 /// An example OptimizerRule that replaces all `col = <const>` predicates with a
 /// user defined function
+#[derive(Default, Debug)]
```

Comment:
this is the end user visible change -- basically you have to `#[derive(Debug)]` if you haven't already